scripts/release_testing_setup.sh

```diff
@@ -22,17 +22,26 @@ if [ -f "${HOME}/.cocoapods/repos" ]; then
   find "${HOME}/.cocoapods/repos" -type d -maxdepth 1 -exec sh -c 'pod repo remove $(basename {})' \;
 fi
 
-mkdir -p "${local_sdk_repo_dir}"
-echo "git clone from github.com/firebase/firebase-ios-sdk.git to ${local_sdk_repo_dir}"
-set +x
-# Using token here to update tags later.
-git clone -q https://"${BOT_TOKEN}"@github.com/firebase/firebase-ios-sdk.git "${local_sdk_repo_dir}"
-set -x
+if [ "$TESTINGMODE" = "release_testing" ]; then
+  mkdir -p "${local_sdk_repo_dir}"
+  echo "git clone from github.com/firebase/firebase-ios-sdk.git to ${local_sdk_repo_dir}"
+  set +x
+  # Using token here to update tags later.
+  git clone -q https://"${BOT_TOKEN}"@github.com/firebase/firebase-ios-sdk.git "${local_sdk_repo_dir}"
+  set -x
+  cd  "${local_sdk_repo_dir}"
+elif [ "$TESTINGMODE" = "prerelease_testing" ]; then
+  git fetch --tags origin main
+  git checkout main
+fi
 
-cd  "${local_sdk_repo_dir}"
 # The chunk below is to determine the latest version by searching
 # Get the latest released tag Cocoapods-X.Y.Z for release and prerelease testing, beta version will be excluded.
 test_version=$(git tag -l --sort=-version:refname --merged main CocoaPods-*[0-9] | head -n 1)
+if [ -z "$test_version" ]; then
```

Comment:
Ideally we could have logic to move the latest **unreleased** CocoaPods tag to tip of main.

Reply:
Yes, totally!